virtual-desktop/src/app/window-manager/mvd-window-manager/shared/keybinding.service.ts

```diff
@@ -0,0 +1,45 @@
+
+
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+
+import { Injectable, EventEmitter } from '@angular/core';
+
+@Injectable()
+export class KeybindingService {
+  public keyupEvent = new EventEmitter<KeyboardEvent>();
+  constructor() {
+    this.keyUpHandler = this.keyUpHandler.bind(this);
+  }
+
+  registerKeyUpEvent() {
+    document.addEventListener('keyup', this.keyUpHandler, true);
+  }
+
+  keyUpHandler(event: KeyboardEvent) {
+    if(event.altKey) {
+      console.log('desktop' + event.which);

```

Comment:
We should probably use the zowe logger here (if the log is still necessary)

Reply:
fixing

Reply:
fixed